### DISCUSSION TITLE
Home assistant e cloudflare

### DISCUSSION THREAD

**Discussion by fassatal** [indomus/forum#2700] [HUB: Home Assistant]
Ciao a tutti,
ho da poco installato Home Assistant Core su Debian12 e volevo poterlo utilizzare da remoto tramite Cloudflare.
Ho un mio sito personale, ho modificato i DNS in cloudflare e ho seguito la guida per la commessione da remoto tramite Docker Run.
Devo avere fatto un errore da qualche parte e mi ha restituito errore 1013 quando provo a collegarmi al mio sito.
Ho rifatto tutto da capo e ovviamente da terminale Debian il sistema mi dice che ho già creato un tunnel con il nome che ho scelto e mi fa creare il tunnel solo con un altro nome.
Mi ha restituito nuovamente errore 1013.

Siccome non sono per niente pratico di Debian suppongo di aver fatto abbastanza casino. Mi potete dire con quale comando posso verificare i tunnel che ho creato e con quale comando posso eliminarli tutti in modo da riprovare un'altra volta?
Grazie!!!

Fabio

**Comment by snakuzzo** [comment]
Ciao.
Innanzi tutto Debian non c'entra niente con Cloudflare.
Poi... la causa dell'errore basta cercarla un attimo su google -> https://developers.cloudflare.com/support/troubleshooting/cloudflare-errors/troubleshooting-cloudflare-1xxx-errors/#error-1013-http-hostname-and-tls-sni-hostname-mismatch

Detto questo...assicurati di non aver configurato Core in https

Inoltre, se proprio vuoi riprovare a farlo (ma non mi sembra questo il caso), non serve creare un "nuovo nome", puoi usare benissimo quello già utilizzato prima, basta solo cancellarlo andando sulla dashboard di Cloudflare e cancellare il cname creato dopo l'abilitazione del tunnel.

**Reply by fassatal** [reply]
Ciao e grazie per la risposta.
Lo so che Debian non c'entra niente con Clouflare.
Era solo per spiegare la mia situazione di partenza.
Ci riproverò.
Mi sai solo dire quale comando da terminale posso utilizzare per cancellare i tunnel che ho "battezzato" con il comando docker run.

Grazie

**Reply by snakuzzo** [reply]
Devi semplicemente rifare la procedura, cancellando prima il CNAME sulla dashboard di CF

**Reply by fassatal** [reply]
Ciao,
ho rifatto la procedura ma questa volta se mi collego al mio sito ho come risultato errore 1033. Ho guardato il link che mi hai girato sopra, ma non ho ben capito cosa devo fare.
Grazie e scusa per l'insistenza

**Reply by snakuzzo** [reply]
Se stai usando l'addon, controlla i log dell'addon appunto. Il 1033 dovrebbe indicare che hai problemi a collegarti a Cloudflare per attivare il tunnel

**Reply by fassatal** [reply]
non sto usando l'addon. Ho fatto tutto seguendo la guida qui per l'installazione su Home Assistant Core su Docker

https://indomus.it/guide/collegarsi-da-remoto-a-home-assistant-via-reti-provider-tradizionali-o-che-usino-nat-via-cloudflare/

**Reply by snakuzzo** [reply]
E' inutile che mi linki le guide.
Comunque controlla i log del container che hai creato.
